pkg/cmd/cli/describe/describer.go

```diff
@@ -433,7 +433,6 @@ func (d *ProjectDescriber) Describe(namespace, name string) (string, error) {
 
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, project.ObjectMeta)
-		formatString(out, "Display Name", project.DisplayName)
```

Comment:
I want to still show the DisplayName as a named field when doing

osc describe project my-project

So this line should remain, but it should be

```
formatString(out, "Display Name", project.Annotations["displayName"])
```